YaST/Repos/openSUSE_ppc_Leap_15.3_Servers.xml.in

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
```

Comment:
we don't need this file, besides armv7, other supported archs should've the same url(no ports), and we don't change the external link in the skelcd-control(again, besides armv7)

https://build.opensuse.org/package/view_file/openSUSE:Leap:15.3/skelcd-control-openSUSE/skelcd-control-openSUSE.spec?expand=1